docs/requirements.txt

```diff
@@ -1,4 +1,4 @@
-git+http://github.com/SymbiFlow/sphinx_materialdesign_theme.git@master#egg=sphinx_symbiflow_theme
+git+http://github.com/SymbiFlow/sphinx_materialdesign_theme.git@66f78b9#egg=sphinx_symbiflow_theme
```

Comment:
Don't use an explicit commit here.

Reply:
I wanted to fix revisions of all crucial dependencies so that when one is updated upstream the documentation build system won't break.

Reply:
Removed explicit revision specification.